src/lcm/argmax.py

```diff
@@ -1,31 +1,43 @@
 import jax
 import jax.numpy as jnp
+from jax import Array
```

Comment:
Would look nicer IMO if we imported both from `jax.typing`, assuming that has the same effect.

Reply:
`Array` is not available in the `jax.typing` module because it is not a type per se. The way I do it here is copied from the [JAX Typing Best Practices](https://jax.readthedocs.io/en/latest/jax.typing.html#jax-typing-best-practices) documentation section.

Reply:
Sorry for the noise!